tiledb/ml/readers/tensorflow.py

```diff
@@ -143,7 +144,7 @@ def _generator(
         y_attribute_names: Sequence[str],
         rows: int,
         batch_size: int,
-        buffer_size: Optional[int],
+        buffer_size: int,
```

Comment:
Why do we remove `Optional` here? User should be able to skip this argument while instantiating the reader and I think in case `None` we already set the `buffer_size` equal to the `batch_size` which is not `Optional`.
Except for that everything else LGTM!

Reply:
It is still optional for the user in `__new__`, it is required for the generators (mypy complained about it)